pre-commit

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
```

Comment:
1. Not everyone works on *nix system. Many users are on Windows or use GitHub UI.

Reply:
Change README.md, mark this step for *nix users.